aws/cloudformation/iam.yml.erb

```
@@ -835,6 +835,90 @@ roles.each do |name, config| %>
     Type: AWS::IAM::ServiceLinkedRole
     Properties: 
       AWSServiceName: logger.cloudfront.amazonaws.com
+
+  # ------------------------------------------------------------
+  # Create one AWS IAM User for each type of marketing sites environment that GitHub Actions will create, with
+  # permissions needed to create/update/delete a marketing sites CloudFormation Stack.
+  # ------------------------------------------------------------
+  <%
+    %w(development test production).each do |environment_type|
+  %>
+  GithubUser<%=environment_type%>:
```

two nits: include the word `marketing` to align with how you named the user, and use the word `deployer` to signify that what this user does (as opposed to other things a github action may want to do, like readonly or execute lambdas/tasks, etc)

```suggestion
  GithubMarketingDeployerUser<%=environment_type%>:
```

(if you edit this, also edit UserName below to match)

---

Agreed! I’ve updated the naming convention.